yb-voyager/src/tgtdb/yugabytedb.go

```diff
@@ -1163,6 +1173,46 @@ func (yb *TargetYugabyteDB) isQueryResultNonEmpty(query string) bool {
 	return rows.Next()
 }
 
+func (yb *TargetYugabyteDB) IsAdaptiveParallelismSupported() bool {
```

Comment:
Similarly here, Ideally we can have these functions as part of target interface.
like this function `IsAdaptiveParallelismSupported` if this will be a part of target interface we can return false for other dbs which will make it clear that Adaptive parallelism is not applicable for them.

Let me know, we can get on a call to discuss these if there are any challenges.